server/admin_management/api/app/main.py

```diff
@@ -61,6 +67,10 @@ def custom_generate_unique_id(route: APIRouter):
     allow_headers=["*"],
     allow_credentials=True,
 )
+app.add_exception_handler(HTTPError, requests_http_error_handler)
+app.add_exception_handler(Exception, unhandled_exception_handler)
+app.add_exception_handler(ValidationError, validation_exception_handler)
+
```

Comment:
Migrate/configure exception_handler same as server/backend. Some tests due to this time changes needs these exception handler.